packages/ui/src/components/va-color-indicator/VaColorIndicator.vue

```diff
@@ -74,6 +92,7 @@ export default defineComponent({
   }
 
   &__core {
+    display: block;
     transition: transform 0.1s linear;
     vertical-align: baseline;
```

Comment:
vertical-align is ignored when display: block. Remove it if it's not needed.

Reply:
Was set cause of div (with display block as default) -> span (with display inline as default). So, it's seems to be ignored before.